src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -1373,25 +1373,227 @@ <h4 class="modal-title">
                             </div>
                           </div>
                           <ul class="list-group" id="responseCommentTable-0-1-1-GRQ" style="margin-top:15px;">
-                            <li class="list-group-item list-group-item-warning" id="responseCommentRow-${comment.id}">
-                              <div id="commentBar-${comment.id}">
+                            <li class="list-group-item list-group-item-warning" id="responseCommentRow-1-1-1-1-GRQ">
+                              <div id="commentBar-1-1-1-1-GRQ">
                                 <span class="text-muted">
                                   From: teammates.test@gmail.tmt [Sun, 01 Mar 2026, 11:59 PM UTC] (last edited by teammates.test@gmail.tmt at Mon, 02 Mar 2026, 11:59 PM UTC)
                                 </span>
+                                                     <form class="responseCommentDeleteForm pull-right">
```

Comment:
This line seems to be indented more than necessary. Multiple occurrences in the file.